staging/publishing/rules.yaml

```diff
@@ -330,8 +330,6 @@ rules:
       branch: master
     - repository: apiserver
       branch: master
-    - repository: code-generator
```

Comment:
We still need code-generator here because it is a part of `required-packages`:

https://github.com/kubernetes/publishing-bot/blob/c36ebbc870db789f0b5b6c8cb8c7e47edc012d07/artifacts/scripts/construct.sh#L51-L54

Reply:
@nikhita just this instance?

Reply:
`code-generator` isn't a dependency via go imports (as can be seen from godeps.json) so it needs to be added here. We need code-generator for generating code for API types in sample-apiserver, etc.

Reply:
> @nikhita just this instance?

(sorry, my page didn't refresh :))

it'll be needed for all instances where `code-generator` is a required-package but not in godeps.json.

Reply:
i dropped all the cleanup for code-generator and left just the missing dependency.